src/dapp/dapp.interface.ts

```diff
@@ -61,20 +62,23 @@ export interface BroadcastDappMessageCommand {
   title: string;
   message: string;
   notificationTypeId?: string;
+  addressTypes?: AddressTypeDto[];
```

Comment:
Let's use `AddressType` in dapp interface instead of dto, since dto is internal structure that is subject to change